File: src/main/java/ru/javarush/cryptoanalyzer/konovalov/Commands/CaesarEncoder.java
package ru.javarush.cryptoanalyzer.konovalov.Commands;

import ru.javarush.cryptoanalyzer.konovalov.entity.Result;

public class CaesarEncoder implements CaesarAction {
    @Override
    public Result execute(String[] parameters) {
        String mainText = parameters[0];
        String encryptedText = parameters[1];
        int key = Integer.parseInt(parameters[2]);

        return applyCipherKey(mainText, encryptedText, key);
    }
}

File: src/main/kotlin/commands/CaesarEncoder.kt
package commands
import entity.Result

class CaesarEncoder : CaesarAction {
    override fun execute(parameters: Array<String>) : Result{
        val mainText = parameters[0]
        val encryptedText = parameters[1]
        val key = parameters[2].toInt()

        return applyCipherKey(mainText,encryptedText,key)
    }
}